config/initializers/console.rb

```diff
@@ -71,4 +71,22 @@ def hard_delete_invoice(id)
       puts "Invoice #{id} has been successfully deleted." # rubocop:disable Rails/Output
     end
   end
+
+  def create_organization(org_name:, email:)
+    organization = Organizations::CreateService
+      .call(name: org_name, document_numbering: "per_organization")
+      .raise_if_error!
+      .organization
+
+    result = Invites::CreateService.call(
+      current_organization: organization,
+      email: email,
+      role: :admin
+    )
+
+    url = "https://app.getlago.com/invitation/#{result.invite.token}"
```

Comment:
I think for the URL you need to use the ENV var to be sure to have the url of the current cluster (and use it locally).
Maybe the URL could be added to the service too and have `result.invite_url`.